devops/openshift/api-deploy.yml

```diff
@@ -232,19 +232,19 @@ objects:
                 httpGet:
                   path: /health/
                   port: "${{PORT}}"
-                initialDelaySeconds: 40
+                initialDelaySeconds: 30
                 periodSeconds: 30
-                timeoutSeconds: 30
+                timeoutSeconds: 1
               livenessProbe:
                 failureThreshold: 3
                 successThreshold: 1
                 httpGet:
                   path: /health/
                   port: "${{PORT}}"
                   scheme: HTTP
-                initialDelaySeconds: 10
+                initialDelaySeconds: 30
                 periodSeconds: 30
-                timeoutSeconds: 15
+                timeoutSeconds: 1
```

Comment:
Can we follow the same timeouts from queue-consumers on this one?
The DB check timeout is already 1s, which means that it could take up to 1s to reply.
Please make it 3s for both probes.